bigquery_etl/query_scheduling/utils.py

```diff
@@ -55,3 +55,45 @@ def is_schedule_interval(interval):
     or a timedelta []d[]h[]m
     """
     return SCHEDULE_INTERVAL_RE.match(interval)
+
+
+def schedule_interval_delta(schedule_interval1, schedule_interval2):
+    """Return the time delta between two schedule intervals as timedelta string."""
```

Comment:
This probably doesn't work for all possible cases, but I think for the ones we currently have in our DAGs it should work fine. I'll open an issue to investigate this further later.